### ISSUE TITLE
Wetkit Updates

### ISSUE THREAD

**Issue by ssari** [wet-boew/wet-boew-drupal#136]
Congratulations all for the excellent work in the WetKit Drupal distribution.

The distribution has been deployed to the Policy Horizons Canada’s website, and I would like to update the profile, Webexp (April/12) to Wetkit, before our project dead line next week. So, I wonder when we should expect to have back features such as the Slider and drop-down Mega-menu.

Thank you all for updating jQuery and SASS, as well as, the Wiki documentation that is much helpful to us.

Solange

**Comment by sylus** [comment]
All of those features are in already :) I would love to assist in updating you guys :) 

So you know there will be no more refactoring just steady improvements so will only have to update once. The Apps Model and git pull will then take care of any subsequent updates.

**Comment by ssari** [comment]
Thanks William,

I'm still getting used to the panels and layouts, so problably I didn't get the correct configuration.

Your help is appreacited. It wa my guess:
Slider:
   added View:Slider to the Top region 
   set display: Mater
  none in Configure view Slider(Master)
  set pane style: Wetkit Nav  - h3, span-2, id=gcwu-connect.
  --> display slides vertically

Mega-menu
  added menu-col in each sub-level
  Multilingual options ??? this is new
- Translation mode: Translate and Localize ....
- Language: English
  save.
  --> French and English items got all mess up.

Please forgive me the misunderstood; I though the slides were being tested yet.

Would you please give a lead to implement the "archived page" message.

Regards, 

Solange.

**Comment by ssari** [comment]
I have reinstalled the wetkit today, and I was able to 
1. Configure the Slider:
    diplay: block
    settings: link title ...
    style: default pane ...
2. Create one level in the mega-menu, but I still have problems with Multilingual - Menu.
    Independently of the option in the translation mode, all items - English and French, are displayed.

Is it any installation problem? The new installation created 161 table in the database, 3 more than previous one.

Does someone else have the same menu problem ?

 Solange Sari 

Ottawa - ON

solange_sari@hotmail.com
